core/rawdb/accessors_chain.go

```diff
@@ -1338,10 +1338,6 @@ func ReadDBSchemaVersion(tx kv.Tx) (major, minor, patch uint32, ok bool, err err
 	return major, minor, patch, true, nil
 }
 
-func WriteLastNewBlockSeen(tx kv.RwTx, blockNum uint64) error {
```

Comment:
can also delete `ReadLastNewBlockSeen` and `kv.LastNewBlockSeen` as they are unused